ui/ui-components/types/page.ts

```diff
@@ -0,0 +1,9 @@
+import {
```

Comment:
I could probably move this to `utils/pageHelper`. Thoughts?

Reply:
Since this is a single file to export one thing, probably worth it.